### ISSUE TITLE
Add support to custom layout

### ISSUE THREAD

**Issue by isDOSe** [rappasoft/vault#19]
Hi Anthony.. Can you add support to add custom layout?.. It is too hard??..

We can use or not use the vault views... but, what if we want to use your vault views with some extra features or design...

Sorry for my english... thanks.

pS. this is not an issue, this is an enhancement request.

**Comment by rappasoft** [comment]
Hmm like helpers to just include the forms or something?

**Comment by isDOSe** [comment]
Exactly... For example, i have my own backend design and i want to add your vault forms.. It would be amazing for us because we have your excelent vault functionality with our custom design.

**Comment by rappasoft** [comment]
Sure i'll try to see how others do it and add it to my todo list.

**Comment by isDOSe** [comment]
Awesome!, Thanks

**Comment by kjghja** [comment]
Hi i installed this packages..
if User login as a staff he can only see uers List .. Staff can't do edit update or delete and can't see other pages how can i do that..
